### PR TITLE
Add "ParameterURL" metadata field

### PR DESCRIPTION
**Pull request by Lestropie**
One aspect of the "model fit parameters" vs. "model-derived parameters" distinction is that sometimes an individual can propose taking the outcomes of some model fitting process and compute some novel derivative measure from those data, and the relevant reference for that derived parameter may be different to that of the model itself. Eg. The original 1994 DTI paper doesn't present FA.

With #92, metadata relating to "the model" is encapsulated within its own metadata dictionary, and there a URL can be provided for the model. With this PR I propose that in some instances it may be desired to provide a URL that describes just the specific parameter that is encoded in the corresponding data file.

### CODE REVIEW

File: src/derivatives/05-diffusion-derivatives.md [hunk 1]
```diff
@@ -393,7 +394,8 @@ Dictionary `"ResponseFunction"` has the following reserved keywords:
                 "FitMethod": "ols",
                 "OutlierRejectionmethod": "None"
             }
-        }
+        },
+        "ParameterURL": "https://doi.org/10.1002/nbm.1940080707"
```
Comment (arokem) [inline src/derivatives/05-diffusion-derivatives.md:398]: Small quibble, but maybe this one is a better reference? 

```suggestion
        "ParameterURL": "https://doi.org/10.1016/j.jmr.2011.09.022"
```

I believe it's the first reference to the 0-1 _normalized_ FA

**Review by arokem — APPROVED** [review]
I believe that the one comment I have just proves that this is a good idea 😄

File: src/derivatives/05-diffusion-derivatives.md (hunk 1 above)
Comment (Lestropie) [reply src/derivatives/05-diffusion-derivatives.md:398]: I'd always had it in my mind that the right citation for FA was the 1996 one. I was just looking through my Zotero and saw the 1995 one, questioned it, found this, and thought maybe all my citations of the 1996 one were erroneous:

FA_1995

I had not realised that it's different to the 1996 one:

FA_1996

Is the first one then a mathematical error?

**Comment by arokem** [comment]
Oh - interesting! Yes, I think that the 1995 _is_ the correct reference here, and also yes: I think that paper has a math error in it (I see no other explanation). Funny.

**Pull request merged**